apisix/plugins/prometheus/exporter.lua

```diff
@@ -76,7 +76,8 @@ function _M.init()
             {"state"})
 
     metrics.etcd_reachable = prometheus:gauge("etcd_reachable",
-            "Config server etcd reachable from APISIX, 0 is unreachable")
+            "Config server etcd reachable from APISIX, 0 is unreachable",
+            {"hostname"})
```

Comment:
we'd better use a new `metric` like `node_info`.
store the `hostname` in `etcd_reachable`, it makes me confused.